src/v0/util/index.js

```diff
@@ -1351,11 +1351,11 @@ const errorStatusCodeKeys = ['response.status', 'code', 'status'];
  * @param {Number} defaultStatusCode default status code that has to be set
  * @returns {Number}
  */
-const getErrorStatusCode = (error, defaultStatusCode = 400) => {
+const getErrorStatusCode = (error, defaultStatusCode = 500) => {
```

Comment:
Let's create a constant for the default status code and use it.

Reply:
Will address this later once the other PR is merged

Reply:
It's hardly a two-minute job.
Nevertheless, please create a new task card or add this to the scope of any open tasks.
CC @krishna2020